frb/data/Galaxies/181112/FRB181112_host.json

```diff
@@ -26,6 +26,24 @@
         "u-r": 1.1526518377513286,
         "u-r_err": 0.15634427608374452
     },
+    "morphology": {
+        "PA": 1.08,
+        "PA_err": 28.06,
+        "b/a": 0.93,
+        "b/a_err": 0.06,
+        "dec": -52.9709368638997,
+        "dec_err": 0.0126,
+        "mag": 13.55,
```

Comment:
This seems too bright (I have r = 21.73 mag).

Reply:
@SunilSimha ?